proto/storage/auth_provider.proto

```diff
@@ -35,4 +35,14 @@ message AuthProvider {
     }
     // EXPERIMENTAL.
     repeated RequiredAttribute required_attributes = 11;
+
+    // EXPERIMENTAL.
+    // visibility allows to specify whether the auth provider should show up when listing the auth providers
+    // during login.
+    enum Visibility {
+        UNSET                    = 0;
```

Comment:
Do we actually want an `UNSET` value here? Or should `VISIBLE` just be the default (value 0)?

If you keep this, please rename it to `UNSET_VISIBILITY`, `VISIBILITY_UNSET` or the like

Reply:
Used `VISIBLE` as default now, also removed the `DefaultVisible` option now as, by default, the provider will now be already visible.